lib/auth/keygen/keygen.go

```diff
@@ -76,13 +70,10 @@ func New(ctx context.Context, opts ...Option) *Keygen {
 	return k
 }
 
-// Close stops the precomputation of keys (if enabled) and releases all resources.
-func (k *Keygen) Close() {
-	k.cancel()
-}
+// Close is a no-op in order to satisfy sshca.Authority.
+func (k *Keygen) Close() {}
```

Comment:
I think we can also remove this receiver from the interface and this implementation. It looks like this is the only implementation in our whole code base, and it's empty now.